scanner/indexer/indexer_test.go

```diff
@@ -0,0 +1,54 @@
+package indexer
+
+import (
+	"testing"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_parseImageURL(t *testing.T) {
```

Comment:
nit: I think the convention is usually something like `TestParseContainerImageURL[_someSpecificCase](t *testing.T)`